lib/components/molecules/Tabs/Tab/Tab.js

```diff
@@ -30,6 +32,9 @@ const Tab = ({
   isSelected,
   onClick,
   onKeyDown,
+  inheritedStyles,
```

Comment:
If you are considering them as fixes your are wrong. Please revert all changes expect your own components

Reply:
all changes have been roll backed